lib/open_telemetry_decorator/attributes.ex

```diff
@@ -16,26 +16,29 @@ defmodule OpenTelemetryDecorator.Attributes do
     |> Tracer.set_attributes()
   end
 
-  def get(all_attributes, requested_attributes) do
-    Enum.reduce(requested_attributes, [], fn requested_attribute, taken_attributes ->
-      case get_attribute(all_attributes, requested_attribute) do
-        {name, value} -> Keyword.put(taken_attributes, name, value)
-        _ -> taken_attributes
-      end
-    end)
+  def get(all_attributes, requested_attributes, values_as_otlp \\ true) do
+    otlp_attributes =
+      Enum.reduce(requested_attributes, [], fn requested_attribute, taken_attributes ->
+        case get_attribute(all_attributes, requested_attribute) do
+          {name, value} -> Keyword.put(taken_attributes, name, value)
+          _ -> taken_attributes
+        end
+      end)
+
+    if values_as_otlp, do: values_to_otlp(otlp_attributes), else: otlp_attributes
```

Comment:
Prior code had converted target values to be OTLP compliant.
Doing it after the fact now so that we can access real `link` value (which is an opaque tuple)